src/System.Private.CoreLib/shared/System/IO/Path.cs

```diff
@@ -861,7 +861,9 @@ private static string GetRelativePath(string relativeTo, string path, StringComp
             //  C:\Foo\Bar C:\Bar\Bar L3, S2 -> ..\..\Bar\Bar
             //  C:\Foo\Foo C:\Foo\Bar L7, S1 -> ..\Bar
 
-            StringBuilder sb = StringBuilderCache.Acquire(Math.Max(relativeTo.Length, path.Length));
+            Span<char> initialBuffer = stackalloc char[64];
```

Comment:
The other path manipulation functions use 260.

Reply:
Increased